lib/network.js

```diff
@@ -384,6 +385,51 @@ function _setDefaults(config) {
 }
 
 
+function _applyOverrides(config) {
+
+  function applyOverrideValue(value, fn) {
+    if (value) {
+      fn(value);
+    }
+  }
+
+  function applyEnvOverride(name, fn) {
+    applyOverrideValue(process.env[name], fn);
+  }
+
+
+  _.forEach(config.overrides, function(override) {
+
+    const flag = override.env && override.env.flag;
+    if (flag && process.env[flag]) {
+
+      applyEnvOverride(override.env.port, function(value) {
+        console.info(`override ${override.name}: port = ${value}`)
+        config.edgemicro.port = value;
+      })
+
+      if (override.headers) {
+
+        applyOverrideValue(override.headers.retarget, function(value) {
+          console.info(`override ${override.name}: retarget_header = ${value}`)
+          config.edgemicro.retarget_header = value;
+        })
+
+      }
+
+      applyOverrideValue(override.log_to_console, function(value) {
```

Comment:
this seems like something that should go into the top level `edgemicro.logging` object to begin with.